### DISCUSSION TITLE
Figure out a way to bulk-merge all the open PRs

### DISCUSSION THREAD

**Discussion by vinitshahdeo** [vinitshahdeo/inspirational-quotes#2021] [Ideas]
### Hello 👋  

I'm glad you're here.

I never expected so [many PRs](https://github.com/vinitshahdeo/inspirational-quotes/pulls) to pile up here, and of course, manually merging them is not an option. I'm creating this thread to brainstorm ideas for efficiently merging the open PRs. 💪.

#### **A GitHub Action (or maybe a script) to automate the bulk merge would be really handy. Your ideas are much appreciated.** Let's discuss.

Thank you very much!

🙏

**Comment by gokulrajpuviyarasu** [comment]
**why This project is excluded from the Hacktoberfest ?**

**Reply by vinitshahdeo** [reply]
I guess, due to the high volume of open PRs as it's not feasible to mark them closed manually.

**Comment by kimku003** [comment]
I haven't read the whole project yet, but looking at the json file of the quotes, they're only in English... If this is really the case in the application, I'd like to configure it to be available in other languages...
